api/v1alpha1/ratelimit_types.go

```diff
@@ -161,6 +168,15 @@ type HeaderMatch struct { // TODO: zhaohuabing this type could be replaced with
 	// +optional
 	// +kubebuilder:validation:MaxLength=1024
 	Value *string `json:"value,omitempty"`
+
+	// Invert specifies whether the value match result will be inverted.
+	// Do not set this field when Type="Distinct", implying matching on any/all unique
```

Comment:
Can we add a CEL validation for this?

Reply:
hey @zhaohuabing can you provide some hints/examples for @rudrakhp to try out ?
these CEL validations can be validated in https://github.com/envoyproxy/gateway/blob/main/test/cel-validation/backendtrafficpolicy_test.go